frontend/src/Services/WardService.ts

```diff
@@ -0,0 +1,12 @@
+import * as API from "./APIService";
+import { Ward } from "../datatypes/ListView.types";
+
+export const getWardTypes = async () => API.APIGET(`/api/wardTypes`);
+
+export const addWard = async (ward: Partial<Ward>) => {
+  return API.APIPOST(`/api/wards`, ward);
```

Comment:
Similar to jendrik's comment, maybe assign api/wards to a variable so that we only have to change it in one place if it changes.

Reply:
Fixed in 4dd10cb